pkg/odo/cli/pipelines/wizard.go

```diff
@@ -90,6 +123,59 @@ func (io *WizardParameters) Complete(name string, cmd *cobra.Command, args []str
 	return nil
 }
 
+func checkBootstrapDependencies(io *WizardParameters, kubeClient kubernetes.Interface, spinner status) error {
+
+	var errs []error
+	client := utility.NewClient(kubeClient)
+	log.Progressf("\nChecking dependencies\n")
+
+	spinner.Start("Checking if Sealed Secrets is installed at kube-system namespace", false)
```

Comment:
How about "...is installed with the default configuration"?

Reply:
I've updated the spinner messages